wolfssl/wolfcrypt/cryptocb.h

```diff
@@ -454,7 +454,7 @@ typedef struct wc_CryptoInfo {
         void *ctx;
     } cmd;
 #endif
-#if HAVE_ANONYMOUS_INLINE_AGGREGATES
+#ifdef HAVE_ANONYMOUS_INLINE_AGGREGATES
```

Comment:
I think you should be able to simplify this by removing the `ifdef`s around the `union` and just putting in a `WOLF_ARG_DUMMY_MEMBER` at the end of the structure. Then you'll always have the benefit of a packed structure with the `union`s in place.

Reply:
that's extraneous to this PR -- only updated because of the refactor from `#if` to `#ifdef`.